Tasks/PublishSymbols/task.json

```diff
@@ -37,25 +37,23 @@
             "label": "Search Pattern",
             "defaultValue": "**\\bin\\**\\*.pdb",
             "required": false,
-            "helpMarkDown": "The pattern used to discover the pdb files to publish.",
-            "groupName": "advanced"
```

Comment:
It should be required because the task will not work without it and I don't consider this advanced.  It is a normal part of that task that would be used 100% of the times you use this task